specification/hugr.md

```diff
@@ -1915,3 +1917,35 @@ differences include:
   - I note re. closures that MLIR expects the enclosing scope to make
     sure any referenced values are kept ‘live’ for long enough. Not what
     we do in Tierkreis (the closure-maker copies them)\!
+
+### Appendix 2: Node types and their edges
+
+The following table shows which edge kinds may adjoin each node type.
+
+Under each edge kind, the inbound constraints are followed by the outbound
+constraints. The symbol ✱ atands for "any number". For example, "1, ✱" means
+"one edge in, any number out".
+
+Except for `Module` it is assumed that these are non-root nodes. The `Module`
+row of the table applies to root nodes of any type.
+
+| Node type      | `Value` | `Order` | `Static` | `ControlFlow` | `Hierarchy` | Children |
+| -------------- | ------- | ------- |--------- | ------------- | ----------- | -------- | 
+| `Module`       | 0, 0    | 0, 0    | 0, 0     | 0, 0          | 0, ✱        |          |
+| `Def`          | 0, 0    | 0, 0    | 0, ✱     | 0, 0          | 1, ✱        | DSG      |
+| `Declare`      | 0, 0    | 0, 0    | 0, ✱     | 0, 0          | 1, 0        |          |
+| `AliasDef`     | 0, 0    | 0, 0    | 0, 0     | 0, 0          | 1, 0        |          |
+| `AliasDeclare` | 0, 0    | 0, 0    | 0, 0     | 0, 0          | 1, 0        |          |
+| `Const`        | 0, 0    | 0, 0    | 0, ✱     | 0, 0          | 1, 0        |          |
+| `LoadConst`    | 0, 1    | 1, 1    | 1, 0     | 0, 0          | 1, 0        |          |
```

Comment:
Could introduce "+" to mean "1 or more" and use it for here? And outgoing order edges are any number right?
```suggestion
| `LoadConst`    | 0, 1    | 1, *    | 1, 0     | 0, 0          | 1, 0        |          |
```

Reply:
and 1 out to the Output

Reply:
OK